docker-cloudwatch-exporter.tf

```diff
@@ -0,0 +1,66 @@
+resource "github_repository" "cloudwatch_exporter" {
+  name             = "cloudwatch-exporter"
+  description      = "cloudwatch-exporter"
+  auto_init        = true
+
+  allow_merge_commit     = false
+  delete_branch_on_merge = true
+  has_issues             = true
+
+  lifecycle {
+    prevent_destroy = true
+  }
+
+  template {
+    owner = "${var.github_organization}"
+    repository = "dataworks-repo-template-docker"
+  }
+}
+
+resource "github_team_repository" "cloudwatch_exporter_dataworks" {
+  repository = "${github_repository.cloudwatch-exporter.name}"
```

Comment:
should be underscores (repeats in several places)

Reply:
As mentioned had not raised it yet, in fact your comment comes after the fix had been put in place